src/components/forms/editSiteForm/index.tsx

```diff
@@ -10,16 +10,29 @@ import styled from 'styled-components';
 
 interface EditSiteFormProps {
   readonly formInstance: FormInstance<EditSiteRequest>;
+  readonly onEdit: (formLat: number, formLng: number) => void;
 }
 
 const EditForm = styled(Form)`
   width: 100%;
 `;
 
-const EditSiteForm: React.FC<EditSiteFormProps> = ({ formInstance }) => {
+const EditSiteForm: React.FC<EditSiteFormProps> = ({
+  formInstance,
+  onEdit,
+}) => {
   return (
     <>
-      <EditForm name="basic" form={formInstance}>
+      <EditForm
+        name="basic"
+        form={formInstance}
+        onValuesChange={(changedValues, allValues) => {
+          if (changedValues.lat || changedValues.lng) {
+            // @ts-ignore
```

Comment:
❓Does typescript show an error without this line?

Reply:
yeah 😢

Reply:
I like how I get a warning on that too :(
![image](https://user-images.githubusercontent.com/33704204/214205493-52ae9edc-726a-4b9f-9d88-63dc0ddc73c5.png)

Could we just do this?

```suggestion
            const values = allValues as EditSiteRequest;
            onEdit(new google.maps.LatLng(values.lat, values.lng));
````